Sources/Plasma/PubUtilLib/plAvatar/CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ target_link_libraries(
         plMessage
         plModifier
         plPhysical
+        pfCCR # FIXME
```

Comment:
This is a terrible solution obviously - I left it in here so I can ask about it. The `plAvatarMgr::WarpPlayer...` methods use the `plCCRError` enum from pfCCR as a status return value, causing this backwards dependency from PubUtilLib on FeatureLib.

The obvious solution would be to move the `plCCRError` enum into one of the PubUtilLib modules, but I don't know which one would be appropriate. Alternatively, we could simply stop using this enum and just return success/failure from the `WarpPlayer...` methods. Especially considering that nothing actually checks the return values currently, and apparently these methods are the only real user of pfCCR in the entire open-source codebase...

Reply:
Fixed now by making the `WarpPlayer` methods return just `hsOK`/`hsFail` and no specific error codes. I now also removed `plCCRError` (and the remaining near-empty CCR code) because nothing else used it.